CONTRIBUTING.md

```diff
@@ -0,0 +1,193 @@
+# Contributing to Numberscope
+
+## If you are new to software development, watch the video tutorials
+
+Watch the video series on contributing to Numberscope
+[here](https://www.youtube.com/playlist?list=PLA4KIQBQQRb5ccOdr9v0iLw_fKHup1PkU).
+This video series will introduce you to Numberscope, GitHub, the software we
+use in the Numberscope project, and the workflow for contributing to
+Numberscope.
+
+## If you have software dev experience, here's the TL;DR
+
+1. Create a fork of the Numberscope repository you'd like to contribute to.
+2. Create a branch within that repository.
+3. Do your work on the branch you created. (Make sure your pre-commit hooks
+   are working.)
+4. When you think your code is in a good state, submit a pull request.
```

Comment:
what are pre-commit hooks?  is that mentioned anywhere else?  are there important things to say?